nilearn/input_data/nifti_labels_masker.py

```diff
@@ -49,13 +50,26 @@ class NiftiLabelsMasker(BaseMasker, CacheMixin):
         See http://nilearn.github.io/manipulating_images/input_output.html
         Region definitions, as one image of labels.
 
+    labels : list of str, optional
+        Full labels corresponding to the labels image. This is used
+        to improve reporting quality if provided.
+        Warning: The labels must be consistent with the label
+        values provided through `labels_img`.
+
     background_label : number, optional
-        Label used in labels_img to represent background. Default=0.
+        Label used in labels_img to represent background.
+        Warning: This value must be consisent with label values and
+        image provided.
+        Default=0.
 
     mask_img : Niimg-like object, optional
         See http://nilearn.github.io/manipulating_images/input_output.html
         Mask to apply to regions before extracting signals.
 
+    resolution : float, optional
```

Comment:
What is this ?

Reply:
@bthirion In order to compute the volume in mm^3 for each region I need to know what is the resolution of the atlas provided. I'm not sure there is another way to get this information than having the user providing it.

Reply:
No, you get that from the image affine (we have code in Nilearn that already does that).

Reply:
@bthirion Absolutely! Sorry for my stupid comment.